bin/prettier.js

```diff
@@ -232,6 +232,19 @@ if (stdin) {
 
     let output;
 
+    if (argv["list-different"]) {
+      try {
+        if (!prettier.check(input, options)) {
+          console.log(filename);
+          process.exitCode = 1;
+        }
+      } catch (e) {
+        // should never hit this block since check catches errors before it returns
```

Comment:
Ha, fair. I'm just used to providing for IE8. There's a pretty slim chance that this will be run in IE8. :D